include/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.hpp

```diff
@@ -19,14 +20,27 @@ namespace ostk
 {
 namespace physics
 {
+
+namespace environment
+{
+namespace object
+{
+class Celestial;
+}
+}  // namespace environment
+
 namespace coordinate
 {
+
+class Position;
```

Comment:
Sure you need to forward declare here instead of just #include?

Reply:
You have to forward declare due to circular dependencies unfortunately. Similar to how we declare Frame in Position.